### DISCUSSION TITLE
LeverFi (LEVER)

### DISCUSSION THREAD

**Discussion by ethvv** [ethvv/v#671] [Binance Research]
<h1 align="center">LeverFi (LEVER)</h1>
<p align="center"><em>一个提供杠杆现货交易的DeFi平台。</em></p>
<p align="center"><em>2022.6.23</em></p>
<hr/>
<ul class="jsx-2167690016"><li>LeverFi是一个DeFi平台，可以进行杠杆现货交易。它的前身是<strong><a href="https://medium.com/rampdefi/ramp-reinventing-2022-96b8f7e900df" target="_blank" rel="noopener,noreferrer" class="jsx-2057463090 ">Ramp</a></strong>。</li><li>交易者存入抵押物后就能以高达10倍的杠杆率进行交易。交易者可以将抵押物存入挖矿协议，从而在杠杆交易的同时获得挖矿收益。</li><li>LEVER是项目的原生代币。目前LEVER的用例包括：<ul class="jsx-2167690016"><li><strong><a href="https://docs.leverfi.io/lever-token-utility" target="_blank" rel="noopener,noreferrer" class="jsx-2057463090 ">治理</a></strong>：持有者可以将LEVER锁定6-48个月，以获得xLEVER代币。xLEVER可以用于治理投票。锁定期越长，产生的xLEVER就越多。</li><li><strong><a href="https://docs.leverfi.io/lever-token-utility" target="_blank" rel="noopener,noreferrer" class="jsx-2057463090 ">质押</a></strong>：用户可以用xLEVER代币入股，以获得协议奖励的份额。</li></ul></li><li>该项目由以下几个主要部分组成，共同工作。<ul class="jsx-2167690016"><li><strong><a href="https://docs.leverfi.io/components/collateral-manager" target="_blank" rel="noopener,noreferrer" class="jsx-2057463090 ">抵押品管理模块</a></strong>：抵押品管理模块管理抵押品的存款、提款、估价和挖矿收益。</li><li><strong><a href="https://docs.leverfi.io/components/lending-manager" target="_blank" rel="noopener,noreferrer" class="jsx-2057463090 ">借贷管理模块</a></strong>：借贷管理模块管理借贷资产的存入、提取、估价和挖矿收益。</li><li><strong><a href="https://docs.leverfi.io/components/trading-manager" target="_blank" rel="noopener,noreferrer" class="jsx-2057463090 ">交易管理模块</a></strong>：交易管理模块管理交易的执行、结算和路由。</li><li><strong><a href="https://docs.leverfi.io/components/risk-manager" target="_blank" rel="noopener,noreferrer" class="jsx-2057463090 ">风险管理模块</a></strong>：风险管理模块计算账户健康状况、贷款价值比以及其他与风险有关的参数，从而控制LeverFi的总风险。</li></ul></li><li>截至2022年6月23日，LEVER代币的总供应量为35,000,000,000。目前的流通供应量为16,800,000,000（48% 的总代币供应量）。</li><li>在<strong><a href="https://medium.com/rampdefi/ramp-reinventing-2022-96b8f7e900df" target="_blank" rel="noopener,noreferrer" class="jsx-2057463090 ">这里</a></strong>了解更多关于LEVER的代币分布。</li></ul>

**Comment by MuhammadImaz** [comment]
Can I help?
